botany/src/main/java/binnie/botany/EnumHelper.java

```diff
@@ -0,0 +1,17 @@
+package binnie.botany;
+
+import binnie.core.ModId;
+import binnie.core.util.I18N;
+import binnie.botany.api.IBotanyColored;
+import net.minecraft.util.text.TextFormatting;
+
+public class EnumHelper {
+    public static String getLocalisedName(final IBotanyColored enumClass, boolean withColor) {
```

Comment:
I think it may be better to have some specific handling for the different types of enums in this class, so all the localization logic is kept internally here.